### ISSUE TITLE
Authorize sdk causing error when uploading to the App Store

### ISSUE THREAD

**Issue by DrishyaOB** [AuthorizeNet/accept-sdk-ios#35]
Hi,

As I was trying to upload my application in which I have been using accept-ios SDK for payment, I came across many problems. Below mentioned is the error description.

**ERROR ITMS-90171: "Invalid Bundle Structure - The binary file '<ProjectName>.app/Frameworks/AcceptSDK.framework/libswiftRemoteMirror.dylib' is not permitted. Your app can’t contain standalone executables or libraries, other than the CFBundleExecutable of supported bundles. Refer to the Bundle Programming Guide at https://developer.apple.com/go/?id=bundle-structure for information on the iOS app bundle structure."

ERROR ITMS-90206: "Invalid Bundle. The bundle at '<ProjectName>.app/Frameworks/AcceptSDK.framework' contains disallowed file 'Frameworks'."**

When I searched for solutions, I couldn't find anything regarding Authorize.net, but there were similar cases for other 3rd parties and that is the reason for posting this issue.

I have done a workaround for the issue and also tested in both the development and production scenario in which it seemed to be working fine. But not sure whether it is the right way or may come across any hurdle in the future.

**Action points: -
After achieving the build. I checked the path , '<ProjectName>.app/Frameworks/AcceptSDK.framework/ ' in which I found the files (libswiftRemoteMirror.dylib and folder 'Frameworks' )which is causing the issue. 

I deleted the 2 files and done the further proceeding for the App Store submission.**

**Is it the right way to solve this problem?**

**Comment by adavidw** [comment]
Appears to be fixed now. Can you check again?

**Comment by DrishyaOB** [comment]
The application has been approved in the App Store. It is working fine.
@adavidw Did you come across any similar issue?

**Comment by fizzy871** [comment]
That error occurs because in the AcceptSDK project settings "Always Embed Swift Standard Libraries" value set to "YES". When that value set to "NO", swift libraries do not includes into AcceptSDK framework and there will not be "Framework" folder inside AcceptSDK.Framework. After such changes build uploaded into itunesconnect successfully without additional changes

**Comment by ketan-parmar** [comment]
I am also facing multiple issue when uploading my application on app store. I have my project in objective c and i have integrated accept-ios SDK manually, I mean i haven't used pod. Because when i have tried to used accept-ios SDK with pod it was unable to compile. any solution for uploading app to appstore !  
@RakeshVisa

**Comment by Mindbowser** [comment]
@ketan-parmar any luck? @adavidw any thoughts?

**Comment by Mindbowser** [comment]
@DrishyaOB we are also facing same issue, can you tell us how you solved it?

**Comment by DrishyaOB** [comment]
@Mindbowser I had encountered with 2 issues as mentioned below:

**ERROR ITMS-90171: "Invalid Bundle Structure - The binary file '.app/Frameworks/AcceptSDK.framework/libswiftRemoteMirror.dylib' is not permitted. Your app can’t contain standalone executables or libraries, other than the CFBundleExecutable of supported bundles. Refer to the Bundle Programming Guide at https://developer.apple.com/go/?id=bundle-structure for information on the iOS app bundle structure."

ERROR ITMS-90206: "Invalid Bundle. The bundle at '.app/Frameworks/AcceptSDK.framework' contains disallowed file 'Frameworks'."**

**Action points: -
After achieving the build. I checked the path , '.app/Frameworks/AcceptSDK.framework/ ' in which I found the files ("libswiftRemoteMirror.dylib" and folder 'Frameworks' )which was causing the issue.

I deleted those  and proceeded to the App Store submission.**

I used Xcode 8 for submitting the application.

Good Luck!
